Python/ds/Queue.py

```diff
@@ -0,0 +1,37 @@
+# Stack implementation using List in Python
```

Comment:
It should be "Queue implementation using List in Python".

Reply:
Sorry for that! I have changed it.